source/dotnet/AdaptiveCards.Visualizer.Wpf.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
```

Comment:
do we need pipeline changes here, or is that going to be a subsequent PR?

Reply:
we will need pipeline change; setting pack command with versioning and publishing path and etc.